model/jpa/src/main/java/org/keycloak/models/jpa/JpaRealmProvider.java

```diff
@@ -289,14 +291,25 @@ public RoleModel getClientRole(ClientModel client, String name) {
     public Map<ClientModel, Set<String>> getAllRedirectUrisOfEnabledClients(RealmModel realm) {
         TypedQuery<Map> query = em.createNamedQuery("getAllRedirectUrisOfEnabledClients", Map.class);
```

Comment:
I'm not too sure how the client augmentation is supposed to work with these named queries. If we were to have a client type with associated redirect uris, they wouldn't necessarily be returned in the `Set` here because the query is not client type aware. @mposolda what do you think?

Reply:
Yes, I think you are correct.

In theory, if client has a type, it should add `redirectUris` from that particular client type. However I don't consider this as a blocker for this PR due the:
- This method `getAllRedirectUrisOfEnabledClients` is used only for deprecated functionality (deprecated method `RedirectUtils.getValidateRedirectUris(KeycloakSession session)`)
- I am not yet 100% sure how to treat "collection" properties like `redirectUris` with client types. For instance, should the final `redirectUris` should be union of redirectUris declared on client itself together with those declared on client type? Or should client override the ones declared on client type?

Until we clarify this, I am fine with keep as it is in this PR.